addons/mail/static/src/new/core/thread_model.js

```diff
@@ -6,6 +6,8 @@ import { Partner } from "./partner_model";
 export class Thread {
     /** @type {import("@mail/new/core/follower_model").Follower[]} */
     followers = [];
+    /** @type {import("@mail/new/core/channel_member_model").channelMember[]} */
+    channelMembers = [];
```

Comment:
not really important but we try to sort them alphabetically when we think about it